web/src/Web.App/Controllers/SchoolComparatorsCreateByController.cs

```diff
@@ -11,14 +12,15 @@
 namespace Web.App.Controllers;
 
 [Controller]
-//todo: reinstate once DSI permissions resolved [SchoolAuthorization]
+[SchoolAuthorization]
```

Comment:
Doesn't required school authorisation, just authorisation.

An authenticated user can create a user defined comparator set for any school, not just there own.